src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,13 +1,15 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar" style="display: none"></div>
```

Comment:
IMO, you should revert this change. Changing the way this div is added is not necessary for your bugfix